### DISCUSSION TITLE
I don´t understand how can i use this node

### DISCUSSION THREAD

**Discussion by gedasnow04** [alexkn/node-red-contrib-dlna#15] [Q&A] [answered]
Hi , i installed this node , and i don't know can i use for do dlna in a Samsung tv with dlna 
It would be possible to give an example of how to use this node?

Thanks a lot

**Comment by gedasnow04** [comment]
how can i do for send a video of youtube to my tV ? 
It's possible an example 
Thanks

**Comment by alexkn** [comment, marked as answer]
I actually only use it with radio streams.
But it should also work with videos.

Select your TV as Device in the node configuration.
And send a payload like this to the node:
```json
{
  "action":"load",
  "url":"http://url"
}
```
